test/python/circuit/test_parameters.py

```diff
@@ -1113,6 +1113,7 @@ def test_circuit_with_ufunc(self):
         theta = Parameter(name="theta")
 
         qc = QuantumCircuit(2)
+        qc.p(numpy.abs(phi), 0)
```

Comment:
Would that be more interesting to check?
```suggestion
        qc.p(numpy.abs(-phi), 0)
```